index.html

```diff
@@ -3064,41 +3135,43 @@ <h3>Verify Proof</h3>
 the `proof` value removed.
           </li>
           <li>
-Let <var>transformedData</var> be the result of
-<a href="#dfn-transformation">transforming</a> the <var>unsecuredDocument</var>
-according to a <a>transformation algorithm</a> associated with the
-<a>cryptographic suite</a> specified in <var>proof</var> and the
-<var>options</var> parameters provided as inputs to the algorithm.
-The type of <a>cryptographic suite</a> is specified by the
-<var>proof.type</var> value and MAY be further described by
-<a>cryptographic suite</a>-specific properties expressed in <var>proof</var>.
+Let <var>transformedData</var> be the result of running |suite|'s'
```

Comment:
```suggestion
Let <var>transformedData</var> be the result of running |suite|'s
```

Reply:
Oops, yes, I think this was a mistake. Not sure how it got in there. I'll send a separate PR to fix it.